### ISSUE TITLE
Some recent or most used emojis in ActionSheet for quick reaction to messages.

### ISSUE THREAD

**Issue by vaibhavs2** [zulip/zulip-mobile#4565]
### Look at the actionSheet, how beautiful it is looking.

<img src="https://user-images.githubusercontent.com/31806059/112682834-ad613680-8e96-11eb-98cc-b5ec34706bd1.png" height="640"/>

there should be 4-5 recently used  or most used emojis in the ActionSheet for quick reaction to the messages, it will help users to quick react without navigating one more screen

**Comment by harry-dev98** [comment]
hey i had started to figure out how it can be done, can you assign this issue to me?

**Comment by vaibhavs2** [comment]
I am already working on it,

**Comment by gnprice** [comment]
Thanks @vaibhavs2 for filing this! I agree, this would be good UI to have. It'll especially be great to have a quick way to add :+1: , like we do in the webapp.

Chat thread here: https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/Quick.20reactions/near/1156419 with some discussion on specifics.

**Comment by mirabilos** [comment]
Yes, please… the stock iOS emoji picker gives me two dozen or so recent ones first, so I have my cat and other favourites always pretty much upfront, but in Zulip on GrapheneOS or web I always have to search :'(
